requirements/serve.txt

```diff
@@ -1,3 +1,6 @@
 gradio
 protobuf
 tritonclient[grpc]
+timm
+flash-attn
+openai
```

Comment:
We can accept `openai`

Reply:
Let's put `openai` to runtime.txt

Reply:
@ColorfulDick THIS comment is not resolved.

Reply:
Please don't remove "tritonclient[grpc]"